Index: src/main/java/moe/xmcn/catsero/utils/Logger.java
===================================================================
diff --git a/src/main/java/moe/xmcn/catsero/utils/Logger.java b/src/main/java/moe/xmcn/catsero/utils/Logger.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/moe/xmcn/catsero/utils/Logger.java	
@@ -0,0 +1,23 @@
+package moe.xmcn.catsero.utils;
+
+import java.util.List;
+import java.util.logging.Level;
+
+public interface Logger {
+
+    static void log(String from, String msg) {
+        java.util.logging.Logger.getLogger("CatSero").log(
+                Level.INFO,
+                "[" + from + "]" + msg
+        );
+    }
+    static void log(String from, List<String> msg) {
+        for (var i = 0; i <= msg.toArray().length; i++) {
+            java.util.logging.Logger.getLogger("CatSero").log(
+                    Level.INFO,
+                    "[" + from + "]" + msg.toArray()[i]
+            );
+        }
+    }
+
+}
